swift/aperture/Recorder.swift

```diff
@@ -36,7 +36,10 @@ public class Recorder: NSObject, AVCaptureFileOutputRecordingDelegate {
     }
 
     self.output = AVCaptureMovieFileOutput();
-    self.output?.movieFragmentInterval = CMTimeMake(1,1); // write data to file every 1 second
+
+    // write data to file every 2 seconds
+    // writing every 1 second causes intermittent errors
+    self.output?.movieFragmentInterval = CMTimeMakeWithSeconds(2, 1);
```

Comment:
This is a great workaround. The fact this works alludes to the problem being buffer related. Maybe we should investigate buffering filesystem writes in a more comprehensive manner.

cc @matheuss